src/system/WakeEvent.h

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Derived from `SystemSockets.h`; see https://gist.github.com/kpschoedel/234612fb3cc3a7ec4ff0f16fea731525 for diff